API.md

```diff
@@ -30,6 +30,24 @@ General optional parameters:
     debug:      1
 ```
 
+### Request Example
+
+Making a request to PlayPy using the [curl](https://curl.haxx.se/) tool is straightforward
+
+```
```

Comment:
If you set the language to `ShellSession` it will do some highlighting for you:

\`\`\`ShellSession